UnityProject/Assets/Scripts/UI/Systems/EndRound/RoundEndScoreScreen.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
what about this one? I don't see any linq in your new code

Reply:
yup, removed.